pysus/ftp/databases/sia.py

```diff
@@ -66,8 +66,10 @@ def describe(self, file: File) -> dict:
 
     def format(self, file: File) -> tuple:
         if file.extension.upper() in [".DBC", ".DBF"]:
-            year, month = file.name[-4:-2], file.name[-2:]
-            group, _uf = file.name[:-6].upper(), file.name[-6:-4].upper()
+            digits = ''.join([d for d in file.name if d.isdigit()])
+            chars, _ = file.name.split(digits)
```

Comment:
simply ignores the suffix, it should return all the files with that ends with 'a', 'b' or 'c' (or any other suffix actually) in the format result